web/pom.xml

```diff
@@ -1303,6 +1319,288 @@
       </dependencies>
     </profile>
 
+    <!-- specific to geOrchestra build -->
+    <profile>
+      <id>debianPackage</id>
+      <properties>
+        <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
+      </properties>
+      <build>
+        <finalName>geonetwork</finalName>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <configuration>
+              <classifier>generic</classifier>
+              <archiveClasses>false</archiveClasses>
+              <attachClasses>false</attachClasses>
+              <webResources>
+                <resource>
+                  <directory>${project.basedir}/../schemas</directory>
+                  <includes>
+                    <include>**/src/main/plugin/*</include>
+                  </includes>
+                  <targetPath>WEB-INF/data/config/schema_plugins</targetPath>
+                </resource>
+                <resource>
+                  <directory>${project.basedir}/../web-ui/src/main/resources</directory>
+                </resource>
+                <resource>
+                  <directory>${build.webapp.resources}</directory>
+                </resource>
+              </webResources>
+              <packagingExcludes>
+                xml/schemas/**,
+                catalog/lib/style/bootstrap/docs/**,
+                catalog/lib/style/bootstrap/fonts/**,
+                catalog/lib/style/bootstrap/grunt/**,
+                catalog/lib/style/bootstrap/test-infra/**,
+                catalog/lib/style/font-awesome/css/**,
```

Comment:
i think this line above should be dropped, because with it the `font-awesome/css` subdir isnt present in the `georchestra-generic.war` archive, and thus all icons coming from font-awesome are broken at runtime in the war

(note that i dunno how font-awesome is supposed to work/be packaged, that's just my analysis of the console errors im seeing)

Reply:
#269 has my suggestions, this gives a better debian package